projects/novo-elements/src/elements/picker/Picker.ts

```diff
@@ -108,6 +109,8 @@ export class NovoPickerElement implements OnInit {
   autoSelectFirstOption: boolean = true;
   @Input()
   overrideElement: ElementRef;
+  @Input()
+  maxlength: number;
```

Comment:
You could do: `@Input() maxlength: number = 150;` so that it defaults to 150 if the input is not used.

Reply:
fixed